packages/cli/lib/utils.js

```diff
@@ -266,3 +266,32 @@ exports.renameEJS = function() {
 
   return renameStream;
 };
+
+/**
+ * Get a validate function for object/array type
+ * @param {String} type 'object' OR 'array'
+ */
+exports.validateStringObject = function(type) {
```

Comment:
`function(type: string)` here?

Reply:
CLI is a `JS` project not a `TS` project so no types here. -- In the future we should definitely consider switching to TS.